jmclient/jmclient/taker.py

```diff
@@ -208,6 +208,42 @@ def initialize(self, orderbook):
             self.latest_tx = None
             self.txid = None
 
+        # calculate fidelity bond values and add them to orders
```

Comment:
I'd feel a lot better about this being moved to its own method that is called here. It makes it easier to review later changes (and also the code in general) and ensures the code is not somehow entangled with the surrounding code, which may to some extend prevent issues such as #822